tools/ci_build/github/azure-pipelines/win-ci-pipeline.yml

```diff
@@ -67,26 +67,6 @@ stages:
         WITH_CACHE: true
         MachinePool: 'Win-CPU-2019'
 
-- stage: x64_release_xnnpack
```

Comment:
do we need to disable the whole Windows build? building with different compilers can expose more issues with the code. we would also lose the check that future code changes don't break a Windows build.

if the issue is a test failure, can we temporarily disable the problematic test(s)?

Reply:
It is about memory allocation, which affects all tests. I agree most build pipelines can expose more problems, but if we just pile problems up, it doesn't help.

Reply:
hm, temporarily disabling all tests and still compiling would be better than disabling the whole thing IMO. it might at least prevent new compilation issues from being introduced.

Reply:
Sounds good.

Reply:
I will update this PR accordingly